### ISSUE TITLE
Unnecessary smart quotes in css comments breaks project source code compile

### ISSUE THREAD

**Issue by bronius** [amsul/pickadate.js#1082]
Admittedly, I don't know enough about our build process' gruntfile concat_sourcemap step to give more specifics, but I know that in some environments this step chokes on pickadate/lib/themes/classic.css.  When I replace all smart quotes and apostrophes with ... dumb? ones, the error goes away and build succeeds.

The error shown was:
```
Running "concat_sourcemap:app" (concat_sourcemap) task
    error app/styles/app.scss (Line 5 of vendor/pickadate/lib/themes/classic.css: Invalid US-ASCII character "\xE2")
```

I'll follow with a PR.

**Comment by DanielRuf** [comment]
Which smartquotes for example?

Can you share your setup, config and gruntfile with us?

**Comment by bronius** [comment]
Hi - Thanks for your lightning quick response!  Again, I don't know enough to share the specifics with you, but the relevant part in gruntfile is:
```
  grunt.registerTask('buildScripts', filterAvailable([
                     'jshint:app',
                     'jshint:tests',
                     'coffee',
                     'emberscript',
                     'copy:javascriptToTmp',
                     'transpile',
                     'concat_sourcemap'
                     ]));
```
specifically the `concat_sourcemap` step.  I'd guess there is some concat_sourcemap config (ignore comments from css, allow UTF-8, etc), but I don't know what to change where.  I only know what helped me over this hump.  On the build server, I tried various fixes like environment language variables `"en_US.UTF-8"` and adding `@charset "UTF-8";` to my main app.scss, but nothing else worked for me.

**Comment by bronius** [comment]
tldr; :nothingtodohere:

Hi @DanielRuf Just an update here: I still don't know enough to make this a worthwhile closure, but there's no specific great sense in cluttering your queue.  My issue was environmental: Our build server had empty locale and language settings.  My resolution is:
```
export LC_ALL=en_US.UTF-8
export LANG=en_US.UTF-8
```

Now when I run npm test (which does a build including jshint and I guess some css preprocessing which I couldn't find), my tests pass just fine.  The same was when I had stripped those smart quotes out of the comments in your css.  Fwiw, they're in there: You can see it when viewing the lib/themes/classic.css file right in github, so that part of it is not on my end.  Incidentally, before these locale/language settings, vi/vim couldn't render the smart quotes either.

Thanks for your earlier attention on this :D  Cheers.
